packages/eslint-config-react-app/index.js

```diff
@@ -315,5 +341,37 @@ module.exports = {
     'flowtype/define-flow-type': 'warn',
     'flowtype/require-valid-file-annotation': 'warn',
     'flowtype/use-flow-type': 'warn',
+
+    // https://github.com/testing-library/eslint-plugin-jest-dom
+    ...(hasJestDom
+      ? {
+          'jest-dom/prefer-checked': 'error',
+          'jest-dom/prefer-empty': 'error',
+          'jest-dom/prefer-enabled-disabled': 'error',
+          'jest-dom/prefer-focus': 'error',
+          'jest-dom/prefer-required': 'error',
+          'jest-dom/prefer-to-have-attribute': 'error',
+          'jest-dom/prefer-to-have-text-content': 'error',
+        }
+      : null),
+
+    // https://github.com/testing-library/eslint-plugin-testing-library
+    ...(hasTestingLibrary
```

Comment:
Similarly if these are specific to testing-library APIs, could we always enable and avoid the dependency check? Any harm?